src/structures/gdt.rs

```diff
@@ -149,6 +168,25 @@ impl GlobalDescriptorTable {
         unsafe { lgdt(&self.pointer()) };
```

Comment:
We could call `load_unchecked` here instead of duplicating the `lgdt(&self.pointer())` line.